src/Http/HttpPlugHttpAdapterClient.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Rs\VersionEye\Http;
+
+use GuzzleHttp\Psr7\LazyOpenStream;
+use GuzzleHttp\Psr7\MultipartStream;
```

Comment:
those both are the blocker...php-http doesnt offer multipart streams or file streams yet, see https://github.com/php-http/message-factory/pull/30